src/crt.c

```diff
@@ -0,0 +1,54 @@
+
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include "php_aws_crt.h"
+
+ZEND_DECLARE_MODULE_GLOBALS(awscrt);
+
+PHP_INI_BEGIN()
+STD_PHP_INI_ENTRY("awscrt.log_level", "", PHP_INI_ALL, OnUpdateLongGEZero, log_level, zend_awscrt_globals, awscrt_globals)
```

Comment:
Weird to ask user for a number, vs a string like "Trace".
Are numeric log levels a standard PHP thing?

If this is just you bootstrapping, go ahead and ship it

Reply:
This is me being lazy as hell and just trying to get bootstrapped without any dependencies yet ;)